pkg/cmd/roachtest/roachtestutil/mixedversion/README.md

```diff
@@ -5,15 +5,17 @@ The `mixedversion` package provides an API that teams can use to test behaviour
 * not every node in the cluster is running the same binary. One example of this would be a 4-node cluster where nodes 1 and 2 are running v22.2.7, and nodes 3 and 4 are running v23.1.2.
 * every node in the cluster is running the same binary version, but the cluster version is still at some previous version due to the use of [`cluster.preserve_downgrade_option`](https://www.cockroachlabs.com/docs/stable/upgrade-cockroach-version.html#step-3-decide-how-the-upgrade-will-be-finalized). For example, an operator ran `SET CLUSTER SETTING cluster.preserve_downgrade_option = '22.2'`, then restarted every node with a v23.1.4 binary.
 * every node in the cluster is running the same binary version _and_ the upgrade is happening in the background (i.e., relevant migrations are running). Also known as _finalizing_ the upgrade. For example: in the scenario above, the operator ran `RESET CLUSTER SETTING cluster.preserve_downgrade_option`.
+* in [multi-tenant](#deployment-modes) deployments: the storage cluster is running a given cluster version, but the tenant is still in the previous verison.
 
 ### High Level Overview
 
 Every run of a roachtest that uses the `mixedversion` API follows this high-level structure:
 
 * cluster is initialized at a certain previously released version (possibly but not necessarily the immediate predecessor of the current binary);
+* if the test is running in a _multi-tenant_ deployment, the test tenant is created at some point after the cluster is initialized, but before any user functions run.
 * nodes are stopped and restarted with a different, newer binary. User functions (_hooks_ -- see below) run at arbitrary points during this process. The order in which nodes are restarted is unspecified;
-* a downgrade _may_ happen -- that is, nodes are once again restarted, and the newer binary is replaced by an older binary. This simulates the need for the operator to perform a downgrade in case the cluster cannot be upgraded for some reason. In case a downgrade is performed, the cluster will set the `cluster.preserve_downgrade_option` cluster setting accordingly prior to any node being upgraded;
-* eventually, every node in the cluster is restarted with the most recent binary (i.e., the one passed to roachtest with the `--cockroach` flag). The upgrade is then finalized.
+* a downgrade (or rollback) _may_ happen -- that is, nodes are once again restarted, and the newer binary is replaced by an older binary. This simulates the need for the operator to perform a downgrade in case the cluster cannot be upgraded for some reason. In case a downgrade is performed, the cluster will set the `cluster.preserve_downgrade_option` cluster setting accordingly prior to any node being upgraded;
+* eventually, every node in the cluster is upgraded all the way to most recent binary (i.e., the one passed to roachtest with the `--cockroach` flag). That upgrade is then finalized.
```

Comment:
Nit: "...upgraded to the most recent binary..."

Reply:
Nit: "That upgrade it then finalized (i.e., downgrade is no longer allowed)."

Reply:
Fixed and added the extra context, thanks.